apps/www/lib/database.types.ts

```diff
@@ -1,112 +1,1524 @@
-export type Json = string | number | boolean | null | { [key: string]: Json } | Json[]
+export type Json = string | number | boolean | null | { [key: string]: Json | undefined } | Json[]
 
 export interface Database {
   public: {
```

Comment:
im not sure we should be pulling in the entire schema here.

Reply:
Yep, i think we remove the tables we don't need.

Reply:
You're right, I removed all tables beside `partners` and partner_contacts`. I rebased and force pushed new commits.